app/config/default_parameters.yml

```diff
@@ -30,4 +30,7 @@ parameters:
     cache_redis_port: 6379
 
     # Content types included in Recommendations from YooChoose account
-    ez_recommendation.default.recommender.included_content_types: [blog, article, place]
\ No newline at end of file
+    ez_recommendation.default.recommender.included_content_types: [blog, article, place]
+
+    #Location id of the root for form uploads
```

Comment:
As a 'eZ developer', `root` is related to the root of the content tree.
I propose `#Location id of the content item for form uploads`